planetiler-custommap/src/main/java/com/onthegomap/planetiler/custommap/configschema/TagCriteria.java

```diff
@@ -0,0 +1,59 @@
+package com.onthegomap.planetiler.custommap.configschema;
+
+import static com.onthegomap.planetiler.expression.Expression.matchAnyTyped;
+import static com.onthegomap.planetiler.expression.Expression.matchField;
+
+import com.onthegomap.planetiler.custommap.TagValueProducer;
+import com.onthegomap.planetiler.expression.Expression;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+
+public class TagCriteria {
```

Comment:
Now that this isn't a part of the schema, could we just move this static utility into `custommap` so it's near the other similar utilities?